### ISSUE TITLE
Newlines not trimmed in shortened message if message already short

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#123]
```
What steps will reproduce the problem?
Send the message:
sms:Someone:Hello
there.

What is the expected output?
Sending SMS to Someone (xxxxxxx): "Hello there."

What do you see instead?
Sending SMS to Someone (xxxxxxx): "Hello
there."

Compare this with for instance:

sms:Someone:Hello
there. How are you today? Good? Great!

where it is trimmed properly.

Additional informations:
GTalkSMS version = 2.0
Android OS version = 2.2
Phone model = HTC Hero
```

Original issue reported on code.google.com by `Eckankar` on 10 Mar 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 10 Mar 2011 at 5:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Changeset 4a84348ff4 - 
http://code.google.com/p/gtalksms/source/detail?r=4a84348ff4edb0bc6f40bf8a25133f
1a4b6c3975
```

Original comment by `fschm...@gmail.com` on 10 Mar 2011 at 9:46
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.1
```

Original comment by `Florent....@gmail.com` on 14 Mar 2011 at 10:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
